files/main.py

```diff
@@ -55,81 +82,91 @@ def login(username, password):
 		else:
 			LDAP_SEARCH_FILTER = environ["LDAP_SEARCH_FILTER"]
 
-		# Optional parameter
-		LDAP_REQUIRED_GROUPS = ""
-		if "Ldap-Required-Groups" in request.headers:
-			LDAP_REQUIRED_GROUPS = request.headers["Ldap-Required-Groups"]
-		elif "LDAP_REQUIRED_GROUPS" in environ:
-			LDAP_REQUIRED_GROUPS = environ["LDAP_REQUIRED_GROUPS"]
+		# List of groups separated by comma
```

Comment:
It would be nice to have all the retrieval of parameters/headers inside a function. Otherwise the main its a bit too long until you get to the part of the validation logic

Reply:
The object request.headers exists only in some context of the application, also I would like to set these parameters outside in one place..